Improving invariant section to state which function are being checked (#264)
Certora/Documentation: docs/cvl/invariants.md

While verifying an invariant, the Prover checks two things.  First, it checks
that the invariant is established after calling any constructor.  Second, it checks

> that the invariant holds after the execution of any methods, assuming

Reviewer: ```suggestion
that the invariant holds after the execution of any method, assuming
```
Only a single method will be executed before the invariant

Reviewer: Do you think we should clarify that we only refer to `public` (or `external`) methods?

Author: Right - I added and clarified it.

that it held before the method was executed (if it does hold, we say the method

> *preserves* the invariant). By default, an invariant fill be checked for all (non-view) method

Reviewer: ```suggestion
*preserves* the invariant). By default, an invariant will be checked for all (non-`view`) methods
```

on all contracts in the scene - this can be further configured by {ref}`-parametric-contracts`.

> View methods are excluded from invariant checking as by definition they cannot change the state 

Reviewer: ```suggestion
`View` methods are excluded from invariant checking as by definition they cannot change the state 
```

Reviewer: Should we also mention that `pure` methods are skipped? Technically they are view methods as well

> of the contract. 

Reviewer: If we are already clarifying this, "the contract" may be ambiguous, maybe better say "any contract"

Author: I rephrased it.

While verifying an invariant, the Prover checks two things.  First, it checks
that the invariant is established after calling any constructor.  Second, it checks
that the invariant holds after the execution of any methods, assuming that it held 
before the method was executed (if it does hold, we say the method *preserves* the invariant). 

> By default, an invariant fill be checked for any `public` or `external` 

Reviewer: ```suggestion
By default, an invariant will be checked for any `public` or `external` 
```
